ScamSearcher/settings.py

```diff
@@ -127,3 +125,27 @@
         },
     },
 ]
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'file': {
```

Comment:
__[flake8]__
* [F403] 'from settings_local import *' used; unable to detect undefined names